### DISCUSSION TITLE
Welcome to kanri Discussions!

### DISCUSSION THREAD

**Discussion by trobonox** [kanriapp/kanri#5] [Announcements]
## 👋 Welcome!
I'm using Discussions as a place to connect with people that are interested in this project. Here you can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, just open a new discussion with the topic you want to address!

**Comment by 3PSY0N** [comment]
Just a comment to thank you for this tool!

I've used a lot of kanban tools but this one is fast, simple and offline (which is what I needed).

The roadmap shows a lot of good things to come 💪

Thank you

**Reply by trobonox** [reply]
Thank you very much for your feedback! As a solo developer working on this open-source project in my free time I am very glad to hear this and happy to hear you enjoy the app.

Thanks for the support and if there's any feature wishes you have or any bugs you encounter, you are always welcome to open an issue. An alternative way to reach me is the email trobo@kanriapp.com if you want to tell me anything personally.

Best regards
- Trobo

**Comment by dusanvlajkovic** [comment]
Dear @trobonox,

I too wish to thank you for building this tool, I have started using it on a daily basis.

Kind regards from Belgrade,
Dusan
